common.lic

```diff
@@ -66,7 +66,7 @@ module DRC
     clear
     put message
     timer = Time.now
-    while (response = get?) || (Time.now - timer < 15)
+    while (response = get?) || (Time.now - timer < 5)
```

Comment:
❌ It looks like a unit test needs to be updated too

Reply:
Test updated.  Also cleaned up the method to be a bit less spammy in practice